### ISSUE TITLE
Could not get state from entity

### ISSUE THREAD

**Issue by finalbillybong** [custom-components/pyscript#465]
After installing and setting up the blueprints for three of my appliances I get the following error in my logs. 

`This error originated from a custom integration.

Logger: custom_components.pyscript.file.pv_excess_control.on_time
Source: custom_components/pyscript/eval.py:1901 
Integration: Pyscript Python scripting (documentation, issues) 
First occurred: 20:57:00 (3 occurrences) 
Last logged: 20:57:00

Could not get state from entity automation.start_dish_washer_solar_excess: name 'automation.start_dish_washer_solar_excess' is not defined
Could not get state from entity automation.start_washing_machine_solar_excess: name 'automation.start_washing_machine_solar_excess' is not defined
Could not get state from entity automation.start_tumble_dryer_solar_excess: name 'automation.start_tumble_dryer_solar_excess' is not defined`

The name of my automation(s) matches the blueprint(s) as required. 

![525097BE-FD52-4C70-A821-24288755291B](https://user-images.githubusercontent.com/24855560/232891705-f82050e6-51ad-4800-acd2-9a72ca553cd5.png)
![8EB9DDB7-A7EA-451E-B464-7D08B5B28C79](https://user-images.githubusercontent.com/24855560/232891723-2ec73656-f393-4009-86ae-d5935ceabe19.png)

**Comment by Th0mg** [comment]
Hi finalbillybong, i ran into same problem. Can you share how you solved the issue?

Best regards, Thomas

**Comment by finalbillybong** [comment]
I ended up writing my own automations to control everything in the end and not using this at all.

**Comment by Th0mg** [comment]
Hi, thanks for the reply.
I found my (your former?) problem for this case. I gave the automation a descriptive name. And set this exactly for the Automation Entity ID of the blueprint.
But under the hood home assistant modifies the resulting name by changeing spaces to underscore and converting letters to lowercase... E.g. "Wallbox excess" –> wallbox_excess. So to say, you setup a "friendly name" and home assistant creates an internal name meeting some criterias.
Stupid mistake of mine as the blueprint describes this in bold letters, but i didn’t realized during setting up my first automation with a blueprint.
Now it works and i am gaining experience. Maybe i end up also writing my own control as i have a component which needs special control. But let's see.
Happy solar production!
